atd-etl/cris_import/cris_import.py

```diff
@@ -83,21 +87,25 @@ def main():
     GRAPHQL_ENDPOINT_KEY = secrets["graphql_endpoint_key"]
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
+    S3_EXTRACT_BUCKET = secrets["s3_extract_bucket"]
+
     local_mode = False
     if bool(glob.glob("/app/development_extracts/*.zip")):
         local_mode = True
 
     zip_location = None
+    database_location = None
     if not local_mode:  # Production
-        zip_location = download_archives()
+        zip_location, database_location = download_s3_archive()
     else:  # Development. Put a zip in the development_extracts directory to use it.
         zip_location = specify_extract_location()
 
     if not zip_location:
         return
 
-    extracted_archives = unzip_archives(zip_location)
-    for archive in extracted_archives:
+    extracted_archives = unzip_archives(zip_location, database_location)
+    for archive_data in extracted_archives:
+        archive = archive_data[1]
```

Comment:
The `unzip_archives` function returns an array of tuples now, instead of just an array of strings representing the locations of the extracted zip files' temp directories. The tuples contain that string of the location on the file system, but also now the ID of the record for that extract in the SQLite DB. We'll use that later on to mark successful imports.